src/Sarif/Baseline/ResultMatching/SarifLogResultMatcher.cs

```diff
@@ -343,5 +344,24 @@ private List<ExtractedResult> ExtractResultsFromRuns(IEnumerable<Run> sarifRuns)
                 }
             }
         }
+
+        private static IList<VersionControlDetails> MergeVersionControlDetails(IEnumerable<Run> runs)
+        {
+            // use HashSet with EqualityComparer to avoid duplicated entries
```

Comment:
>u [](http://example.com/codeflow?start=15&length=1)

Cap and add . at the end

Reply:
I would remove the entire comment. I don't see the difference between comment and the actual code.

Reply:
removed the comments.